osquery/database/plugins/sqlite.cpp

```diff
@@ -139,6 +139,17 @@ Status SQLiteDatabasePlugin::get(const std::string& domain,
   return Status(1);
 }
 
+Status SQLiteDatabasePlugin::get(const std::string& domain,
+                                 const std::string& key,
+                                 int& value) const {
+  std::string result;
+  auto s = this->get(domain, key, result);
+  if (s.ok()) {
+    value = std::stoi(result);
```

Comment:
Please use safeStringTo* here if possible

Reply:
RocksDBDatabasePlugin::get
SQLiteDatabasePlugin::get

Should be changed in the following pull requests(and serialization will be removed)